cms/src/fields/slides.js

```diff
@@ -7,13 +7,26 @@ export const SLIDE_TYPES = {
   SHARING: 'sharing',
 };
 
+const CONTENT_IMAGE_FIELD = {
+  label: 'Image',
+  name: 'image',
+  widget: 'object',
+  collapsed: true,
+  required: false,
+  i18n: false,
+  fields: [{...IMAGE_FIELD, name: 'source', required: false}],
```

Comment:
Maybe we should change the `IMAGE_FIELD` and `VIDEO_FIELD` to be `{ source: '', description: ''}` as default?

Reply:
And for `VIDEO_FIELD`:
{ source: '', description: '', thumbnail: '' }

Reply:
I'll add it now and we can use it later